elixir/apps/web/lib/web/live/actors/show.ex

```diff
@@ -175,6 +178,59 @@ defmodule Web.Actors.Show do
       </:content>
     </.section>
 
+    <.section>
+      <:title>Authentication Tokens</:title>
+
+      <:action :if={is_nil(@actor.deleted_at)}>
+        <.delete_button
+          phx-click="revoke_all_tokens"
+          data-confirm="Are you sure want to sign out actor from all devices?"
+        >
+          Revoke All
+        </.delete_button>
+      </:action>
+
+      <:content>
+        <.table id="tokens" rows={@tokens} row_id={&"tokens-#{&1.id}"}>
+          <:col :let={token} label="TYPE" sortable="false">
+            <%= token.type %>
+          </:col>
+          <:col :let={token} label="IDENTITY" sortable="false">
+            <.identity_identifier account={@account} identity={token.identity} />
+          </:col>
+          <:col :let={token} label="CREATED">
+            <.created_by account={@account} schema={token} />
+          </:col>
+          <:col :let={token} label="LAST SEEN (IP)">
+            <p>
+              <.relative_datetime datetime={token.last_seen_at} />
+            </p>
+            <p :if={not is_nil(token.last_seen_at)}>
+              <.last_seen schema={token} />
+            </p>
+          </:col>
+          <:col :let={token} label="EXPIRES AT">
```

Comment:
Minor UX I noticed with the relative datetimes. If the column values say "in 3 days" then this could just be "EXPIRES".

```suggestion
          <:col :let={token} label="EXPIRES">
```